community/sigs.yaml

```diff
@@ -1,192 +1,240 @@
+  leadership:
+    chairs:
+    - github: harshad16
+      name: Harshad Nalla
+      company: Red Hat
+    tech_leads:
+    - github: harshad16
+      name: Harshad Nalla
+      company: Red Hat
+  meetings:
+  - description: Let's talk Thoth Tech
+    day: Thursday
+    time: "15:00"
+    tz: UTC (Coordinated Universal Time)
+    frequency: weekly
+    url: https://meet.google.com/kxd-axiz-tym
+  contact:
+    liaison:
+      github: goern
+      name: Christoph Görn
+  subprojects:
+  - name: aicoe-ci
+    owners:
+    - https://raw.githubusercontent.com/AICoE/aicoe-ci/master/OWNERS
+  - name: opf-ci-prow
+    owners:
+    - https://raw.githubusercontent.com/thoth-station/thoth-application/master/prow/OWNERS
+  - name: thoth-ops-infra
+    owners:
+    - https://raw.githubusercontent.com/thoth-station/thoth-ops-infra/master/OWNERS
+- dir: sig-stack-guidance
+  name: Stack Guidance
+  mission_statement: >
+    Work on recommending the most effective, performant and secure software stack
+    for user applications and on actively creating prescriptions and security related
+    information
 
-    charter_link: charter.md
-    label: user-experience
-    leadership:
-      chairs:
-        - github: goern
-          name: Christoph Görn
-          company: Red Hat
-      tech_leads:
-        - github: pacospace
-          name: Francesco Murdaca
-          company: Red Hat
-    meetings:
-      - description: Let's talk Thoth Tech
-        day: Thursday
-        time: "15:00"
-        tz: UTC (Coordinated Universal Time)
-        frequency: weekly
-        url: https://meet.google.com/kxd-axiz-tym
-    contact:
-      liaison:
-        github: goern
-        name: Christoph Görn
-    subprojects:
-      - name: jupyterlab-requirements
-        owners:
-          - https://raw.githubusercontent.com/thoth-station/jupyterlab-requirements/master/OWNERS
-      - name: kebechet
-        owners:
-          - https://raw.githubusercontent.com/thoth-station/kebechet/master/OWNERS
-      - name: s2i-thoth
-        owners:
-          - https://raw.githubusercontent.com/thoth-station/s2i-thoth/master/OWNERS
-      - name: thamos
-        owners:
-          - https://raw.githubusercontent.com/thoth-station/thamos/master/OWNERS
-      - name: user-api
-        owners:
-          - https://raw.githubusercontent.com/thoth-station/user-api/master/OWNERS
+  charter_link: charter.md
+  label: stack-guidance
+  leadership:
+    chairs:
+    - github: fridex
+      name: Fridolín Pokorný
+      company: Red Hat
+    tech_leads:
+    - github: fridex
+      name: Fridolín Pokorný
+      company: Red Hat
+    - github: mayaCostantini
+      name: Maya Costantini
+      company: Red Hat
+  meetings:
+  - description: Let's talk Thoth Tech
+    day: Thursday
+    time: "15:00"
+    tz: UTC (Coordinated Universal Time)
+    frequency: weekly
+    url: https://meet.google.com/kxd-axiz-tym
+  contact:
+    liaison:
+      github: goern
+      name: Christoph Görn
```

Comment:
Wouldn't that be @fridex for this SIG?

Reply:
I saw that @goern was the `contact` for all other SIGs, so I assumed this was not something that should be changed SIG-wise